### DISCUSSION TITLE
필요하지 않은 도메인 삭제

### DISCUSSION THREAD

**Discussion by mini0192** [yeojeong-0824/TeamBack#9] [Q&A]
전에 라라님이 말씀 주신 대로 저희 전체적인 코드를 정리해야 할 것 같아요!
일단 로그는 한번 정리했고
필요 없는 도메인 삭제가 필요해 보이는데 Autocomplete와 Image 도메인은 사용하고 있는 건가요?

**Comment by Go-EunLee** [comment]
Autocomplete 는 성빈님께 말씀드려야 할 것 같고 이미지는 게시글 작성할 때 해야 할 것 같은데 저희 db 메모리가 괜찮은가요?? 아무리 압축을 하거나 제한을 해도 이미지를 올리면 메모리가 좀 클 것 같아서요

**Reply by mini0192** [reply]
이미지는 프론트에서 처리하기로 해서 그러면 image 도메인은 사용하지 않는 건가요?

**Reply by Go-EunLee** [reply]
지금은 따로 사용하고 있진 않습니다! 그렇네요 프론트쪽에서 처리하면 괜찮겠네요!

**Reply by mini0192** [reply]
그러면 image는 삭제할게요!
